pkg/cmd/attestation/verification/extensions.go

```diff
@@ -46,5 +51,26 @@ func verifyCertExtensions(attestation *AttestationProcessingResult, tenant, owne
 		}
 	}
 
+	// if issuer is anything other than the default, use the user-provided value;
+	// otherwise, select the appropriate default based on the tenant
+	if issuer != GitHubOIDCIssuer {
+		want = issuer
+	} else {
+		if tenant != "" {
+			want = fmt.Sprintf(GitHubTenantOIDCIssuer, tenant)
+		} else {
+			want = GitHubOIDCIssuer
+		}
+	}
+
+	certIssuer := attestation.VerificationResult.Signature.Certificate.Extensions.Issuer
+	if !strings.EqualFold(want, certIssuer) {
+		if strings.Index(certIssuer, want) == 0 {
```

Comment:
```suggestion
		if strings.Index(certIssuer, want + "/") == 0 {
```

Minor: this would prevent someone from issuing a cert from `https://token.actions.githubusercontent.com.ms` to try to trigger this error message. Minor because all you would be doing is triggering the error message.